ChangeLog

```diff
@@ -1,6 +1,7 @@
 CHANGES
 =======
```

Comment:
What do you mean by stash?.. Please provide meaningful comment. Rebase and update PR.